generic/system/ksp-5g-network-service-scanning.yaml

```diff
@@ -0,0 +1,35 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-5g-network-service-scanning
```

Comment:
`ksp-5g-network-service-scanning` -> `ksp-mitre-5g-network-service-scanning`
Also, file name change a/cly

Reply:
@raviknox these file names are just place holders. The policy name will be generated only from metadata.yaml